src/components/pages/Profile/RenderProfileContainer.js

```diff
@@ -62,7 +62,7 @@ const RenderProfileContainer = props => {
         </Col>
         <Col span={17}>
           <Title level={1}>{userData.name}</Title>
-          <Card title="My Profile">
+          <Card title="My Profile" extra={<EditProfile />}>
```

Comment:
Really good example of researching and then leaning into the component framework we've chosen for use here in Labs - good work!